server/graphql_schemas/views.py

```diff
@@ -3,16 +3,14 @@
 from graphene_django.views import GraphQLView
 from rest_framework.decorators import authentication_classes, permission_classes, api_view
 from rest_framework.settings import api_settings
+from graphene_file_upload.django import FileUploadGraphQLView
+
 from graphql_schemas.utils.helpers import UnauthorizedCalendarError
 
 from graphql.error.located_error import GraphQLLocatedError
 
 
-class DRFAuthenticatedGraphQLView(GraphQLView):
```

Comment:
What's going on here?

Reply:
@andela-sjames Graphene doesn't accept `multipart/form-data` all data sent through a mutation are `application/json` so it wasn't being possible to upload images. I used [graphene-file-upload](https://github.com/lmcgartland/graphene-file-upload) which inherits already from `GraphQLView` so there was no need to include it anymore